src/RTCDataChannel.js

```diff
@@ -53,7 +53,7 @@ module.exports = function (daemon) {
       this.reliable = typeof opts.reliable === 'boolean' ? opts.reliable : true
 
       this.on('error', (err) => {
-        if (/daemon/i.test(err.message)) daemon.emit('error', err)
+        if (/daemon/i.test(err.message)) wrtc.emit('error', err, this)
```

Comment:
Sorry, one last thing. Could you change to just call `wrtc.emit` below, rather than catching errors here? We'll also be able to get rid of the regex matching.